omod/src/main/webapp/locationMapping.jsp

```diff
@@ -16,41 +18,46 @@
 
 <form method="post" onsubmit="fetchLocationMappings()">
     <input name="locationMappings" type="hidden" value="">
-    <table id="locationMappingTable">
-        <thead>
-        <tr>
-            <th><spring:message code="dhisconnector.locationMapping.location"/></th>
-            <th><spring:message code="dhisconnector.locationMapping.orgUnit"/></th>
-        </tr>
-        </thead>
-        <tbody>
-        <c:forEach items="${locations}" var="location">
-            <tr class="evenRow">
-                <td id="location_${location.uuid}" name="${location.uuid}">${location.name}</td>
-                <c:set value="" var="savedOrgUnitUuid"/>
-                <c:forEach items="${locationToOrgUnitMappings}" var="locationToOrgUnitMapping">
-                    <c:if test="${location.id == locationToOrgUnitMapping.location.id}">
-                        <c:set value="${locationToOrgUnitMapping.orgUnitUid}" var="savedOrgUnitUuid"/>
-                    </c:if>
-                </c:forEach>
-                <input name="savedOrgUnitUuidOf_${location.uuid}" type="hidden" value="${savedOrgUnitUuid}">
-                <td>
-                    <select name="orgUnitOf_${location.uuid}">
-                        <option value=" "> </option>
-                        <c:forEach items="${orgUnits}" var="orgUnit">
-                            <c:set value="" var="isSelected"/>
-                            <c:if test="${orgUnit.id == savedOrgUnitUuid}">
-                                <c:set value="selected" var="isSelected"/>
-                            </c:if>
-                            <option ${isSelected} value="${orgUnit.id}">${orgUnit.name}</option>
-                        </c:forEach>
-                    </select>
-                </td>
+    <openmrs:hasPrivilege privilege="View Location Mappings,Manage Location Mappings">
```

Comment:
thanks @Piumal1999  on ensuring that the person who accesses this page has the required privileges as per your first line,do you really need to re check again at this point for the privileges

Reply:
Yeah, you are right @HerbertYiga. I'll remove that line. Thanks for suggesting.

Reply:
Fixed with d673db3